### ISSUE TITLE
Возврат от покупателя. Не считается вес товара.

### ISSUE THREAD

**Issue by gsbelarus** [GoldenSoftwareLtd/gedemin#2105]
Originally reported on Google Code with ID 2105

```
Торговля.
05.Возврат от покупателя. Не считается Брутто вес товара.
```

Reported by `hjamesGS` on 2010-08-25 21:59:07

**Comment by gsbelarus** [comment]
Reported by `gs1994` on 2010-09-04 21:02:11
- **Labels added**: Type-Defect, Priority-Medium, Depot

**Comment by gsbelarus** [comment]
Reported by `E.Maslouskaya` on 2011-02-18 10:38:50
- **Status changed**: `Duplicate`
- **Merged into**: #2233
